javaslang/src/test/java/javaslang/control/TryTest.java

```diff
@@ -64,6 +67,54 @@ public void shouldGetEmpty() {
 
     // -- Try
 
+    // -- andFinally
+
+    @Test
+    public void shouldExecuteAndFinallyOnSuccess(){
+        MutableInteger count = new MutableInteger(0);
+
+        Try.run(() -> count.setValue(0)).andFinally(() -> count.setValue(1));
+
+        assertThat(count.getValue()).isEqualTo(1);
+    }
+
+    @Test
+    public void shouldExecuteAndFinallyTryOnSuccess(){
+        MutableInteger count = new MutableInteger(0);
+
+        Try.run(() -> count.setValue(0)).andFinallyTry(() -> count.setValue(1));
+
+        assertThat(count.getValue()).isEqualTo(1);
+    }
+
+    @Test
+    public void shouldExecuteAndFinallyOnFailure(){
+        MutableInteger count = new MutableInteger(0);
+
+        Try.run(() -> {throw new IllegalStateException(FAILURE);})
+                .andFinally(() -> count.setValue(1));
+
+        assertThat(count.getValue()).isEqualTo(1);
+    }
+
+    @Test
+    public void shouldExecuteAndFinallyTryOnFailure(){
+        MutableInteger count = new MutableInteger(0);
+
+        Try.run(() -> {throw new IllegalStateException(FAILURE);})
+                .andFinallyTry(() -> count.setValue(1));
```

Comment:
\[FindBugs\] INFO: RV: Return value of Try\.andFinallyTry\(Try$CheckedRunnable\) ignored, is this OK in javaslang\.control\.TryTest\.shouldExecuteAndFinallyTryOnFailure\(\)